dts/bindings/sensor/st,lis2duxs12-common.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2023 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+include: sensor-device.yaml
+
+properties:
+  irq-gpios:
+    type: phandle-array
+    description: |
+      DRDY pin
+
+      This pin defaults to active high when produced by the sensor.
+      The property value should ensure the flags properly describe
+      the signal that is presented to the driver.
+
+  drdy-pin:
+    type: int
+    default: 1
+    enum:
+      - 1 # drdy is generated from INT1
+      - 2 # drdy is generated from INT2
+    description: |
+      Select DRDY pin number (1 or 2).
+
+      This number represents which of the two interrupt pins
+      (INT1 or INT2) the drdy line is attached to. This property is not
+      mandatory and if not present it defaults to 1 which is the
+      configuration at power-up.
+
+  accel-range:
+    type: int
+    default: 0
+    description: |
+      Range in g. Default is power-up configuration.
+    enum:
+      - 0 # 2g  (0.061 mg/LSB)
+      - 1 # 4g  (0.122 mg/LSB)
+      - 2 # 8g  (0.244 mg/LSB)
+      - 3 # 16g (0.488 mg/LSB)
```

Comment:
Please use description field instead of comments.
Description is rendered in doc, see https://docs.zephyrproject.org/latest/build/dts/api/bindings/sensor/st%2Clis2dh12-i2c.html#dtbinding-st-lis2dh12-i2c

Reply:
> Please use description field instead of comments. Description is rendered in doc, see https://docs.zephyrproject.org/latest/build/dts/api/bindings/sensor/st%2Clis2dh12-i2c.html#dtbinding-st-lis2dh12-i2c
> 
> Apply everywhere

ok, sorry. I think that I was already told about it, but I forgot it somehow...

Reply:
Done on #62028